Enable src-alias for keystore (#467)
confluentinc/cp-ansible: VARIABLES.md

***

> ### ssl_keystore_alias

Reviewer: Then again I'm worried that if we default the variable to the empty string then the if statement in the code might think its defined, this should be checked

> ### ssl_keystore_alias

Author: ooh - sorry. I totally missed that that was dynamic :D I will adjust that and the jinja dynamic if necessary.

> ### ssl_keystore_alias

Author: I used now  `src_keystore_alias|length` - which should only trigger if the string is > 0.
Do you think that is sufficient? Alternatively one could do a 
`src_keystore_alias.strip()|length` to ensure somebody is not accidentally putting an empty space into the var - but that might crash with an actual existing srcalias containing spaces, right?

> Keystore source alias for host specific certificate if keystore contains more than one certificate. Used with ssl_provided_keystore_and_truststore: true. May set per host, or use inventory_hostname variable eg "{{inventory_hostname}}"

Reviewer 2: ```suggestion
The alias for a host specific certificate, when the keystore contains more than one certificate. Used with ssl_provided_keystore_and_truststore: true. Can be set per host, or use inventory_hostname variable eg "{{inventory_hostname}}"
```
